src/extensions/default/Microsoft.DotNet.UpgradeAssistant.Steps.Packages/PackageAnalyzerResultProvider.cs

```diff
@@ -37,9 +40,9 @@ public class PackageAnalyzerResultProvider : IAnalyzeResultProvider
             _analysisState = null;
         }
 
-        public async Task<bool> IsApplicableAsync(AnalyzeContext analysis, CancellationToken token)
+        public Task<bool> IsApplicableAsync(AnalyzeContext analysis, CancellationToken token)
         {
-            return await Task.FromResult(true);
+            return Task.FromResult(true);
```

Comment:
Why the change to remove async and await from here?

Reply:
we either need to return the task itself or use configureAwait.
since it's a simple true value, you don't need to add await here since the call is normally awaited.

I can revert if you think the other way was better.

Reply:
Yeah lets put it back the way it was, and focus this PR to be only SARIF log improvements. The `IsApplicableAsync` pattern is used in many areas in UA, change here needs to be reflected everywhere, and I would prefer it to be a separate PR.